### DISCUSSION TITLE
Add timestamp in JSON result

### DISCUSSION THREAD

**Discussion by sidali126** [logpai/Drain3#63] [Ideas]
https://github.com/IBM/Drain3/blob/7e340cd1eb8e27b480b0654c77e5a44a77f1331b/drain3/template_miner.py#L129

Hi, 
Can you please add timestamp for the next version?

kind of :

 ` result = {
            "timestamp": int(time.time()),
 `

Regards,
Ali

**Comment by davidohana** [comment]
Hi, can you describe the use case?
Also, you can always add timeatamp yourself to the dict after you get back the result.

**Reply by sidali126** [reply]
Hello and thank you for your reply :)

In the context of IT monitoring system, I want to convert some templates (events) to time series.
Adding this current timpstamp will be a default value if there is no time (timestamp) in the original logs.

Of course we can add it later to Result dict but having it natively help me a lot.

**Comment by davidohana** [comment]
Actually, I don't understand why it is more difficult to add it just after you get the result. It's a single line.  I prefer the result to be minimalistic unless there is a big advantage here, as it might get serialized or network-transferred.

**Reply by sidali126** [reply]
Thanks, we will stick to the original version.
Regards,
Ali
